incubator/ibm-blockchain-chaincode/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
```

Comment:
Templates not namespaced, but namespaced templates used in resources.

Reply:
I think this is fixed now?